src/System.IO.Compression.Brotli/System/Interop/Library.OSX.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.IO.Compression
+{
+    internal static partial class Library
```

Comment:
Should we have separate files for the Library class (which get removed from the project based on the OS) that just have file name strings?

Can we somehow just set the string depending on the OS?